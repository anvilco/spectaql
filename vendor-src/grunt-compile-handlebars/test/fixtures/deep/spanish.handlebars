<h1>{{saludo}}{{puntuacion}} {{ubicacion}}</h1>