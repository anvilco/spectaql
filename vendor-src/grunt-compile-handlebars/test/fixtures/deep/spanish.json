{
  "saludo": "hola",
  "puntuacion": ",",
  "ubicacion": "mundo"
}
