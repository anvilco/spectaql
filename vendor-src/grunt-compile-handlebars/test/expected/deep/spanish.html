<h1>hola, mundo</h1>